src/importsHandler.ts

```diff
@@ -43,6 +46,6 @@ function filterSameImportFromDifferentModules(importCommands: CodeAction[]): Cod
 }
 
 function getImportName({ title }) {
-    let statement = title.match(importStatement) || title.match(existingImportStatement);
+    let statement = importStatements.map(statement => title.match(statement)).find(Boolean);
```

Comment:
`find(Boolean` gets the only element from the list? maybe its a good idea to ensure its a single one? just a thought..

Reply:
find by definition will return single value (first)if exists. the usage of the Boolean function will return the truthy value, it is logically equal to `.filter(a => a)` IMO if you are familiar with the convention it is more readable. WDYT?